tokens/manage/multisig-safe.md

```diff
@@ -312,4 +313,36 @@ To double check that the correct text was set, you can go through the process ag
 
 Congratulations, you've successfully interacted with a smart contract using the Safe!
 
+## Using Moonbeam Safe APIs
+There are APIs available to read from and interact with Moonbeam Safes for Moonbase Alpha, Moonriver, and Moonbeam.
+
+|     Network    |                                           Multisig API Page                                              |
+|:--------------:|:--------------------------------------------------------------------------------------------------------:|
+|    Moonbeam    |  [{{ networks.moonbeam.multisig.api_page }}]({{ networks.moonbeam.multisig.api_page }}){target=_blank}   |
+|    Moonriver   | [{{ networks.moonriver.multisig.api_page }}]({{ networks.moonriver.multisig.api_page }}){target=_blank}  |
+| Moonbase Alpha |  [{{ networks.moonbase.multisig.api_page }}]({{ networks.moonbase.multisig.api_page }}){target=_blank}   |
+
+### Retrieve Safe Information
+You can retrieve information about Safes from the Moonbeam Safe API. From the Safe page, copy the address of your Safe:
+
+![Contract Interaction Call Result](/images/tokens/manage/multisig-safe/safe-30.png)
+
+Now you can use the API:
+
+ 1. Open the corresponding network page from the Multisig API table above
+ 2. Scroll down to the **safes** section and click on the **/safes/{address}/** endpoint section to expand its panel
+ 3. Click the **Try it out** button on the right
+
+![Contract Interaction Call Result](/images/tokens/manage/multisig-safe/safe-31.png)
```

Comment:
There should be an arrow for all of the items, so I would add in the top of your chrome window so you can see the URL... and also so it's consistent with the other screenshots throughout the guide.

Also the arrow for item 2 isn't centered. I can schedule a quick call with you to go over some pointers for working with inkscape 🙂

Reply:
I've made the changes, and have uploaded the source files pngs, cropped pngs, and svgs to the google drive. 

Surprised that I didn't catch the 2 centered. Very ugly =(